src/imitation/algorithms/bc.py

```diff
@@ -209,20 +231,36 @@ def __init__(
         self.observation_space = observation_space
         self.policy_class = policy_class
         self.device = device = utils.get_device(device)
+
+        # SB3's `ActorCriticPolicy` automatically initializes a optimizer which requires
+        # an argument of this type, but we don't use.
+        unused_lr_schedule = ConstantLRSchedule()
```

Comment:
Maybe you should set the LR to something ridiculous (like `None`, or maybe `NaN`) to make errors more obvious?